src/raftstore/store/snap.rs

```diff
@@ -1082,17 +1093,30 @@ pub struct SnapManager {
     // directory to store snapfile.
     core: Arc<RwLock<SnapManagerCore>>,
     ch: Option<SendCh<Msg>>,
+    limiter: Arc<RateLimiter>,
+    pub write_bytes_per_time: usize,
 }
 
 impl SnapManager {
-    pub fn new<T: Into<String>>(path: T, ch: Option<SendCh<Msg>>) -> SnapManager {
+    pub fn new<T: Into<String>>(
+        path: T,
+        ch: Option<SendCh<Msg>>,
+        bytes_per_time: usize,
+        bytes_per_sec: usize,
+    ) -> SnapManager {
         SnapManager {
             core: Arc::new(RwLock::new(SnapManagerCore {
                 base: path.into(),
                 registry: map![],
                 snap_size: Arc::new(RwLock::new(0)),
             })),
             ch: ch,
+            limiter: Arc::new(RateLimiter::new(
+                bytes_per_sec as i64,
+                100 * 1000, // refill_period_us
```

Comment:
use config

Reply:
It's ok to set `refill period us` and `fairness`  by default, https://github.com/facebook/rocksdb/wiki/Rate-Limiter